src/Services/CourseRegistration/CourseRegistration.API/Application/Behaviors/LoggingBehavior.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace CourseRegistration.API.Application.Behaviors
```

Comment:
We are following a convention for our namespace naming. Please use `OpenCodeFoundation.ESchool.Services.` prefix namespace for service namespaces.

Reply:
We can ignore the namespace change. I am resolving this.